project_hr/migrations/15.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Convert m2o to m2m in 'project.task'
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env["project.task"],
+        "project_task",
+        "employee_ids",
+        openupgrade.get_legacy_name("employee_id"),
```

Comment:
You haven't renamed the column previously, but it's not needed:

```suggestion
        "employee_id",
```

Reply:
done! thanks :)